src/main/java/com/ea/restaurant/grpc/GrpcMongoOrderStatusHistoryServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.ea.restaurant.grpc;
+
+import com.ea.restaurant.grpc.MongoOrderStatusHistoryServiceGrpc.MongoOrderStatusHistoryServiceImplBase;
+import com.ea.restaurant.service.impl.MongoOrderStatusHistoryServiceImpl;
+import com.ea.restaurant.util.GrpcMongoOrderStatusHistoryMapper;
+import io.grpc.stub.StreamObserver;
+import net.devh.boot.grpc.server.service.GrpcService;
+
+@GrpcService
+public class GrpcMongoOrderStatusHistoryServiceImpl extends MongoOrderStatusHistoryServiceImplBase {
+
+  private final MongoOrderStatusHistoryServiceImpl mongoOrderStatusHistoryService;
```

Comment:
Why do you point to the impl instead of using the interface? Remember the principles of the DI. Do not use the implementation, use the interface or abstract class instead